src/server.c

```diff
@@ -3869,6 +3871,9 @@ int processCommand(client *c) {
         }
     }
 
+    /* Now that c->slot has been parsed, accumulate the buffered network bytes-in. */
+    clusterSlotStatsAddNetworkBytesIn(c);
```

Comment:
If we can add condition server.cluster_enabled here, if it is easier for us to understand the logic?